app/src/main/java/pm/gnosis/heimdall/ui/safe/overview/SafesOverviewActivity.kt

```diff
@@ -79,6 +91,16 @@ class SafesOverviewActivity : BaseActivity() {
         disposables += layout_safe_overview_add_safe.clicks()
                 .subscribeBy(onNext = { startActivity(AddSafeActivity.createIntent(this)) },
                         onError = Timber::e)
+
+        disposables += viewModel.shouldShowLowBalanceView()
```

Comment:
This will only trigger once, right? and there is no way to refresh. Don't you think it makes sense to pull every x seconds?

Reply:
We can pull every X seconds yeah 👍